Dockerfile

```diff
@@ -8,6 +8,11 @@ FROM golang:${GO_VERSION}-stretch AS builder
 ARG GOARCH=amd64
 ARG GOOS=linux
 
+# Update stretch repositories
```

Comment:
Looks like the upstream project has moved to https://github.com/kubernetes/dashboard/tree/master/modules/metrics-scraper. can we not use that?

Reply:
The new repo has specific k8s support for specific release tag. For now will proceed using these changes.